wiki/Replay/en.md

```diff
@@ -0,0 +1,285 @@
+
+At the backend, one pair of files will be created in the hidden folder, `Data\r`: `.osr` (_osu replay_; the replay file), and `.osg` (accuracy and combo in real-time; used for spectators' replay only) with same encrypted filename.
+It is highly suggested to **not rename the encrypted filename**, and make use of the export `F2` key instead.
+
+Deleting the `.osg` file has no effect (it is a leftover file for spectator's replay; can be safely removed).
+Deleting the `.osr` file will make the replay _lost forever_ (since the replay data is missing and nothing to export).
+Alternatively, deleting the `scores.db` (contains local leaderboard scores and pointers to encrypted replays) _will make all (not exported) replays and local leaderboard scores lost forever_.
+
+To set a name when not logged-in (_Guest_ account), scroll down the result screen to access the _online results screen_ directly after passing the difficulty and type in the name under _Guest player name_.
+In this screen, an alternative to using the export `F2` key is to enable the `Save replay to Replays folder` button located at the upper-right.
+
+Return to the _Song Selection_ and changes will be updated for this replay (if no name was given, it will simply be blank).
+
+### Failed
+
+![Comparison of game over screen between normal and failed replay][Replay failed]
+
+_Left is the typical game over screen. Right is showing the end of a failed replay._
+
+To access this screen, fail the map by draining the healthbar completely, or fail to fill pass 50% of the healthbar before the end of the difficulty in osu!taiko case.
+In the game over screen, press the `F1` key to replay the currently failed play.
+The replay ends when the screen darkens, with osu!logo in the middle, rather than the game over screen.
+
+Due to the non-standard replay screen, use the `Esc` key to return to Song Selection.
+Retrying the play (as in, actual play) of the beatmap in this state is not possible.
+
+It currently works in Solo mode only when given the game over screen, and does not work if [No Fail(NF)][No Fail wikilink]/ [Relax(RL)][Relax wikilink]/[Auto Pilot(AP)][Auto Pilot wikilink] (impossible to fail) or [Perfect(PF)][Perfect wikilink] (forcefully quick-retry instead) game modifier was used.
+
+**This replay will not be saved nor allowed to be exported**.
+To record the failed replay, please use external third-party software for now since osu! has yet to have native export support for failed replay.
+
+If there was a retry streak, it will be lost once entering _Failed_ replay state.
+
+### Multi
+
+![Multi play and replay][Replay Multi]
+
+_Left is the original Multi play and right is an exported Multi replay_
+
+Replays from Multi can only be exported from _Head-to-Head_ or _Team VS_ team mode only.
+The replay will **not be saved internally and not considered in the local leaderboard**.
+Multi elements will not be saved as shown in the image above.
```

Comment:
[Leftmost image] See the **Match Setup** leaderboard at the left and my placement in the Match Setup at the lower-right [the three `...` because it is still preparing]? (Head-To-Head)

Exporting and comparing a Team Vs play would make it much more obvious but eyyyy lazy.